### PR TITLE
[IMP] move button to go to website remove app suggestion buttons - hi…

### PR DESCRIPTION
**Pull request by jao-odoo**
Task: https://www.odoo.com/web#id=32979&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720

pad: https://pad.odoo.com/p/r.0209a7593eac3ca11007a7b3c5d7ab78

### CODE REVIEW

File: addons/website/controllers/backend.py [hunk 1]
```diff
@@ -9,23 +9,15 @@ class WebsiteBackend(http.Controller):
...
-        has_group_system = request.env.user.has_group('base.group_system')
-        has_group_designer = request.env.user.has_group('website.group_website_designer')
-        if has_group_system:
-            apps_data = dict((app['name'], app) for app in request.env['ir.module.module'].sudo().search_read(
-                ['|', ('name', 'ilike', 'website'), ('application', '=', True)],
-                ['id', 'sequence', 'name', 'shortdesc', 'state'],
-                order='sequence ASC'))
-        else:
-            apps_data = {}
+        has_group_system = request.env['res.users'].has_group('base.group_system')
```
Comment (tde-banana-odoo) [inline addons/website/controllers/backend.py]: This change is not necessary. Could you explain why you did it ?

File: addons/website_sale/data/demo.xml [hunk 2]
```diff
@@ -785,6 +785,8 @@ Weight: 31 grams</field>
             <field name="pricelist_id" ref="product.list0"/>
             <field name="team_id" ref="sales_team.salesteam_website_sales"/>
             <field name="date_order" eval="(datetime.now()-relativedelta(days=7)).strftime('%Y-%m-%d %H:%M:%S')"/>
+            <field name="confirmation_date" eval="(datetime.now()-relativedelta(days=7)).strftime('%Y-%m-%d %H:%M:%S')"/>
+            <field name="state">sale</field>
```
Comment (tde-banana-odoo) [inline addons/website_sale/data/demo.xml]: Call methods triggering the business flow. Avoid direct write that bypasses everything.

Comment (tde-banana-odoo) [inline addons/website_sale/data/demo.xml]: Update all demo data you changed accordingly.

File: addons/website/static/src/js/website.backendDashboard.js [hunk 3]
```diff
@@ -116,6 +113,10 @@ var Dashboard = Widget.extend(ControlPanelMixin, {
         });
     },
 
+    render_dashboards_header: function() {
+        this.$('.o_dashboard_common').replaceWith(QWeb.render('website.dashboard_header', {widget: this}));
```
Comment (tde-banana-odoo) [inline addons/website/static/src/js/website.backendDashboard.js]: Wait ... what ? No. Here you replace the content by the content rendered once again. This seems very unnecessary and probably not efficient. Why did you did it ?

File: addons/website/static/src/js/website.backendDashboard.js [hunk 4]
```diff
@@ -140,6 +139,15 @@ var Dashboard = Widget.extend(ControlPanelMixin, {
 
             var tick_values = self.getPrunedTickValues(chart_values[0].values, 5);
 
+            chart.interactiveLayer.tooltip.contentGenerator(function(d) {
+                var date = moment(d.value);
+                return  '<table><thead><tr><td colspan="3"><strong>'+ d.series[0].key +'</strong></td></tr></thead>'+
```
Comment (tde-banana-odoo) [inline addons/website/static/src/js/website.backendDashboard.js]: You should avoid that. Why did you did it ? If necessary embed it in a template somewhere.

File: addons/website/static/src/js/website.backendDashboard.js [hunk 5]
```diff
@@ -226,8 +235,15 @@ var Dashboard = Widget.extend(ControlPanelMixin, {
     },
 
     on_reverse_breadcrumb: function() {
+        var self = this;
         web_client.do_push_state({});
         this.update_cp();
+        this.fetch_data().then(function() {
```
Comment (tde-banana-odoo) [inline addons/website/static/src/js/website.backendDashboard.js]: What is the purpose of that ?

**Comment by hmo-odoo** [comment]
continue work on https://github.com/odoo/odoo/pull/18757

**Pull request closed without merging**